tests/platform_tests/args/cont_warm_reboot_args.py

```diff
@@ -0,0 +1,19 @@
+def add_cont_warm_reboot_args(parser):
+    '''
+    Adding arguments required for continuous warm-reboot test case
+    '''
+    parser.addoption(
+        "--cont_reboot_limit",
```

Comment:
Change limit to count please, to avoid confusion, cont should be expand to continuous.

Reply:
Change added.